<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IcvCurveFilterDialog</class>
 <widget class="QDialog" name="IcvCurveFilterDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>407</width>
    <height>230</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>184</y>
     <width>171</width>
     <height>32</height>
    </rect>
   </property>
   <property name="locale">
    <locale language="English" country="UnitedStates"/>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>87</x>
     <y>117</y>
     <width>291</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>361</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>Select:</string>
   </property>
   <widget class="QRadioButton" name="radioComandName">
    <property name="geometry">
     <rect>
      <x>24</x>
      <y>23</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Name</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioLineId">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>21</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Line id</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioDirection">
    <property name="geometry">
     <rect>
      <x>244</x>
      <y>20</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Direction</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioCompleteComand">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>60</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Complete cmd</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioPromt">
    <property name="geometry">
     <rect>
      <x>24</x>
      <y>60</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Promt</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioPosition">
    <property name="geometry">
     <rect>
      <x>244</x>
      <y>60</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Position</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>27</x>
     <y>120</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>Keywords:</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="previewCheckBox">
   <property name="geometry">
    <rect>
     <x>223</x>
     <y>149</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Preview</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox">
   <property name="geometry">
    <rect>
     <x>143</x>
     <y>150</y>
     <width>72</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>In All</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>IcvCurveFilterDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>IcvCurveFilterDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
